workflows/base-template/template.yaml

```diff
@@ -0,0 +1,180 @@
+  # You can, in fact, remove this from parameters and fix it in the artifacts below. This is just for demo.
+  - name: source
+    value: https://github.com/onepanelio/Mask_RCNN.git
+    displayName: Model source code
+    type: hidden
+    visibility: private
+
+  # Input path for this workflow
+  # The value will be automatically populated by CVAT, if you will run this from CVAT.
+  - name: cvat-annotation-path
+    value: annotation-dump/sample_dataset
+    hint: Path to annotated data in default object storage (i.e S3). In CVAT, this parameter will be pre-populated.
+    displayName: Dataset path
+    visibility: private
+    
+  # Output path for this workflow
+  # The value will be automatically populated by CVAT, if you will run this from CVAT.
+  - name: cvat-output-path
+    value: workflow-data/output/sample_output
+    hint: Path to store output artifacts in default object storage (i.e s3). In CVAT, this parameter will be pre-populated.
+    displayName: Workflow output path
+    visibility: private
+  
+  # Path to checkpoint
+  # Since Onepanel has a filesyncer, all possible values will be displayed in the CVAT
+  - name: cvat-finetune-checkpoint
+    value: ''
+    hint: Select the last fine-tune checkpoint for this model. It may take up to 5 minutes for a recent checkpoint show here. Leave empty if this is the first time you're training this model.
+    displayName: Checkpoint path
+    visibility: public
+  
+  # Number of classes
+  # If you run this from CVAT, it will automatically populate this field based on number of labels in the given CVAT task.
+  - name: cvat-num-classes
+    displayName: Number of classes
+    hint: Number of classes (i.e in CVAT taks) + 1 for background
+    value: 81
+    visibility: private
+    
+  # A set of hyperparameters used by the source code
+  - name: hyperparameters
+    displayName: Hyperparameters
+    visibility: public
+    type: textarea.textarea
+    value: |-
+      stage-1-epochs=1    #  Epochs for network heads
+      stage-2-epochs=2    #  Epochs for finetune layers
+      stage-3-epochs=3    #  Epochs for all layers
+    hint: "Please refer to our <a href='https://docs.onepanel.ai/docs/getting-started/use-cases/computervision/annotation/cvat/cvat_annotation_model#arguments-optional' target='_blank'>documentation</a> for more information on parameters. Number of classes will be automatically populated if you had 'sys-num-classes' parameter in a workflow."
+    
+  # This helps CVAT determine in which format it should dump the data.
+  # Common ones are: cvat_coco, cvat_tfrecord, cvat_yolo
+  - name: dump-format
+    value: cvat_coco
+    displayName: CVAT dump format
+    visibility: public
+      
+  # Docker image to use for this workflow
+  # You can use any public image here (i.e PyTorch)
+  # The jupyterlab image from Onepanel has many dependencies pre-installed
+  - name: docker-image
```

Comment:
I don't see this being used anywhere in the template

Reply:
ahh, forgot to update the var name below. Fixed now.